code/game/machinery/newscaster.dm

```diff
@@ -188,11 +186,12 @@ var/list/obj/machinery/newscaster/allCasters = list()
 	var/obj/item/weapon/photo/photo = null
 	var/channel_name = ""
 	var/c_locked=0
-	var/hitstaken = 0
+	var/health = 60
```

Comment:
so all breakable machines use the same health variable. This will make moving health to the /obj/machinery level easier in the future.